webui/react/src/pages/ExperimentDetails/ExperimentTrials.tsx

```diff
@@ -69,7 +70,7 @@ const ExperimentTrials: React.FC<Props> = ({ experiment, pageRef }: Props) => {
   const [trials, setTrials] = useState<TrialItem[]>();
   const [canceler] = useState(new AbortController());
 
-  const { settings, updateSettings } = useSettings<Settings>(settingsConfig);
+  const { settings, updateSettings } = useSettings<Settings>(configForExperiment(experiment.id));
```

Comment:
suggest to memoize

Reply:
let me know if the last commit is the right way to memoize these, if so then I'll merge